docs/source/json-api/index.rst

```diff
@@ -72,7 +72,9 @@ From a DAML project directory:
       --address <value>
             IP address that HTTP JSON API service listens on. Defaults to 127.0.0.1.
       --http-port <value>
-            HTTP JSON API service port number
+            HTTP JSON API service port number. A port number of 0 will let the system pick up an ephemeral port. Consider specifying port-file option with port number 0.
+      --port-file <value>
+            Optional file where to write the allocated HTTP port number. Used to inform clients in CI about which port HTTP JSON API listens on. Defaults to none, that is file does not get created/updated.
```

Comment:
```suggestion
            Optional file where to write the allocated HTTP port number. This can be used to inform clients in CI about which port the HTTP JSON API listens on. Defaults to none, that is, no file gets created/updated.
```